### PR TITLE
Added composer.json file.

### PR DESCRIPTION
**Pull request by juanlopez4691**
### Description
- Added a composer.json file to be able to install the plugin with Composer. This is required for Flynt/Bedrock based sites.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "authors": [
+        {
+            "name": "Netzstrategen",
+            "email": "hallo@netzstrategen.com",
+            "homepage": "https://netzstrategen.com"
+        }
+    ],
+    "require": {
+        "php": ">=7.0.0",
+        "composer/installers": "^1"
```
Comment (sun) [inline composer.json]: This package doesn't require the installer, only the root project does.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "local-plugins/woocommerce-permalink-structure",
```
Comment (sun) [inline composer.json]: The vendor name should be netzstrategen

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "type": "wordpress-plugin",
+    "authors": [
+        {
+            "name": "Netzstrategen",
```
Comment (sun) [inline composer.json]: netzstrategen always in lowercase when standalone

**Review by sun — CHANGES_REQUESTED** [review]

**Comment by juanlopez4691** [comment]
@sun Applied your suggested fixes. Thanks!

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "type": "wordpress-plugin",
+    "authors": [
+        {
+            "name": "netzstrategen",
+            "email": "hallo@netzstrategen.com",
+            "homepage": "https://netzstrategen.com"
+        }
+    ],
+    "require": {
+        "php": ">=7.0.0"
```
Comment (sun) [inline composer.json]: I think we're actually requiring at least 7.1 in all of our code, which is a big difference due to the null coalesque operator.
```suggestion
        "php": ">=7.1.0"
```

**Review by sun — APPROVED** [review]

File: composer.json (hunk 4 above)
Comment (juanlopez4691) [reply composer.json]: Thanks for your help!

**Pull request merged**